cmd/commands/tx.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	cmn "github.com/tendermint/go-common"
 	client "github.com/tendermint/go-rpc/client"
-	"github.com/tendermint/go-wire"
+	wire "github.com/tendermint/go-wire"
```

Comment:
Sorry about that. Have you used https://godoc.org/golang.org/x/tools/cmd/goimports? It automatically updates your Go import lines and is built in my editor plugin - https://github.com/fatih/vim-go. The problem is that `goimports` uses a simple scheme to detect whenever the particular package is used or not: names (definition and usages) must be equal.

Example:

```
import "github.com/tendermint/go-wire" won't work
import "github.com/tendermint/wire" will work

wire.JSON...
```

Did it makes sense to you?